sdk/core/core/inc/az_span.h

```diff
@@ -260,6 +260,32 @@ AZ_NODISCARD az_result az_span_to_uint64(az_span span, uint64_t* out_number);
  */
 AZ_NODISCARD az_result az_span_to_uint32(az_span span, uint32_t* out_number);
 
+/**
+ * @brief az_span_find searches for `target` in `source`, returning an #az_span within `source` if it finds it.
+ *
+ * @param[in] source The #az_span with the content to be searched on.
+ * @param[in] target The #az_span containing the token to be searched in `source`.
+ * @param[out] out_span The #az_span pointing to the first occurrence of `target` in `source`, if it is found.
```

Comment:
How does using back ticks look like in the generated docs? Elsewhere I have seen folks use `\p` to refer to the parameter.

http://www.doxygen.nl/manual/commands.html#cmdp

Reply:
It renders the text as `code`. So exactly like markdown. I personally prefer it over the doxygen tags because if you only have one work to render in `code` then \p works fine. But for multiple word you need to use `<tt>my text </tt>`. So backticks is consistent whether it be single or multi word.

Reply:
I see. However, `\p` is meant for parameters, which can't be multiple words. I think either is fine, but we should be consistent across the board.

The fact that the same file/type has two different formats just highlights this inconsistency to the reader.

Here are some questions that might help decide which way to  go:
1) `\p` barely formats the parameters differently from regular text (the difference is hard to tell). Is that a good thing or do we want the back ticks which highlight it more prominently?
2) What do folks reading docs generated from doxygen expect and are familiar with?
3) This isn't as important, but since we ship sources, what format do folks generally use in their own projects (i.e. what's common in C projects out there that do use doxygen)?